src/scaling.jl

```diff
@@ -31,8 +31,8 @@ struct MeanStdScaling <: AbstractScaling
     σ::Real
 
     """
-        MeanStdScaling(A::AbstractArray; dims=:, inds=:) -> MeanStdScaling
-        MeanStdScaling(table, [cols]) -> MeanStdScaling
+        MeanStdScaling(A::AbstractArray; dims=:, inds=:, corrected=true) -> MeanStdScaling
```

Comment:
Can we just pass generic `kwargs...`? Or does it get confused about what to do with the `dims` kwarg?